src/remoted/config.c

```diff
@@ -122,6 +126,14 @@ cJSON *getRemoteConfig(void) {
                 }
                 cJSON_AddItemToObject(conn,"denied-ips",list);
             }
+            if (logr.tcp && logr.tcp->connection_overtake_time) {
+                cJSON *tcp = cJSON_CreateObject();
+
+                sprintf(connection_overtake_time,"%d",logr.tcp->connection_overtake_time);
+
+                cJSON_AddStringToObject(tcp, "connection_overtake_time", connection_overtake_time);
```

Comment:
Why don't you add an number to the object instead of a string?

Reply:
Changed!